.github/workflows/ci.yml

```diff
@@ -112,16 +118,16 @@ jobs:
         run: make test_fluvio_install
 
       - name: Install Fluvio CDK
-        run: fluvio install cdk
+        run: fluvio install cdk --develop
```

Comment:
Why `--develop` ?

Reply:
I was just to use cdk latest version. With the new release we can get back to stable version. but maybe it is better to use this version so we can detect earlier issues?